frame/dvm/ethereum/src/lib.rs

```diff
@@ -292,6 +292,24 @@ pub mod pallet {
 			);
 			Self::internal_transact(target, input)
 		}
+
+		/// Internal transaction dispatch call.
+		#[pallet::weight(10_000_000)]
+		pub fn substrate_transact(
```

Comment:
To be replaced by https://github.com/darwinia-network/darwinia-messages-substrate/blob/8c877be1c4078b7a1b7a6361037b8ff9db74d93c/modules/dispatch/src/lib.rs#L323

Reply:
@HackFisher Some clean work has submited. I want to raise separated pr in the `darwinia-messages-substrate` later. So please review this and merge this.